.ci/install_agent.sh

```diff
@@ -12,4 +12,5 @@ tag="$1"
 
 source "${cidir}/lib.sh"
 
-build_and_install "github.com/kata-containers/agent" "" "" "${tag}"
+agent_repo="${agent_repo:-github.com/kata-containers/agent}"
```

Comment:
How about moving `agent_repo=`, `osbuilder_repo=`, *etc* to `lib.sh` since that is sourced by all these scripts? That way they only need to be defined once which would make maintenance easier (useful if this repo ends up moving later to https://github.com/kata-containers/kata-containers for example :)

Reply:
@jodh-intel. I **really** like your idea, do you prefer this as part of this PR or as a follow-up patch?

If you prefer this as part of this PR (that would be my preference), we should add a DNM label to this PR.

Reply:
I'd prefer this PR, but will defer to @chavafg for the final decision.

Reply:
yeah, let's add the change as part of this PR, @wainersm can you please do this change? 
Thanks. 
in the meantime adding DNM as @fidencio suggested, so it does not merge accidentally

Reply:
@chavafg sorry, my time was dragged by other stuffs. I should be working on a new version of this PR soon.

Reply:
One last thing before I make the changes @jodh-intel proposed. The list of repositories are all being set in the ` resolve-kata-dependencies.sh` too. So I am considering to move those statements to `lib.sh` then make it sourced by `resolve-kata-dependencies.sh`.

We will end up with something like this:

```
diff --git a/.ci/lib.sh b/.ci/lib.sh
index 1e75f6f..1d685cc 100755
--- a/.ci/lib.sh
+++ b/.ci/lib.sh
@@ -6,6 +6,16 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+# Repositories needed for building the kata containers project.
+export agent_repo="${agent_repo:-github.com/kata-containers/agent}"
+export proxy_repo="${proxy_repo:-github.com/kata-containers/proxy}"
+export runtime_repo="${runtime_repo:-github.com/kata-containers/runtime}"
+export shim_repo="${shim_repo:-github.com/kata-containers/shim}"
+export tests_repo="${tests_repo:-github.com/kata-containers/tests}"
+export packaging_repo="${packaging_repo:-github.com/kata-containers/packaging}"
+export osbuilder_repo="${osbuilder_repo:-github.com/kata-containers/osbuilder}"
+export katacontainers_repo="${katacontainers_repo:-github.com/kata-containers/kata-containers}"
+
 export KATA_RUNTIME=${KATA_RUNTIME:-kata-runtime}
 export KATA_KSM_THROTTLER=${KATA_KSM_THROTTLER:-no}
 export KATA_NEMU_DESTDIR=${KATA_NEMU_DESTDIR:-"/usr"}
diff --git a/.ci/resolve-kata-dependencies.sh b/.ci/resolve-kata-dependencies.sh
index fa98e06..1ee37c4 100755
--- a/.ci/resolve-kata-dependencies.sh
+++ b/.ci/resolve-kata-dependencies.sh
@@ -7,15 +7,9 @@
 
 set -e
 
-# Repositories needed for building the kata containers project.
-agent_repo="${agent_repo:-github.com/kata-containers/agent}"
-proxy_repo="${proxy_repo:-github.com/kata-containers/proxy}"
-runtime_repo="${runtime_repo:-github.com/kata-containers/runtime}"
-shim_repo="${shim_repo:-github.com/kata-containers/shim}"
-tests_repo="${tests_repo:-github.com/kata-containers/tests}"
-packaging_repo="${packaging_repo:-github.com/kata-containers/packaging}"
-osbuilder_repo="${osbuilder_repo:-github.com/kata-containers/osbuilder}"
-katacontainers_repo="${katacontainers_repo:-github.com/kata-containers/kata-containers}"
+cidir=$(dirname "$0")
+source /etc/os-release || source /usr/lib/os-release
+source "${cidir}/lib.sh"
 
 apply_depends_on() {
        # kata_repo variable is set by the jenkins_job_build.sh
```

Reply:
@wainersm - Thanks - sounds good to me.

Reply:
While moving things in the same place, can you order them alphabetically :-)